src/assets/i18n/fr.json5

```diff
@@ -4887,53 +5560,100 @@
   "submission.sections.upload.form.group-label": "Groupe",
 
   // "submission.sections.upload.form.group-required": "Group is required.",
-  "submission.sections.upload.form.group-required": "Groupe requis.",
+  "submission.sections.upload.form.group-required": "Le groupe est requis.",
 
   // "submission.sections.upload.form.until-label": "Grant access until",
-  "submission.sections.upload.form.until-label": "Accès accordé jusque",
+  "submission.sections.upload.form.until-label": "Accorder l'accès jusqu'au",
+
+  // "submission.sections.upload.form.until-hint": "Select the date until which the related access condition is applied",
+  "submission.sections.upload.form.until-hint": "Sélectionner la date jusqu'à laquelle l'autorisation d'accès liée est appliquée.",
 
   // "submission.sections.upload.form.until-placeholder": "Until",
-  "submission.sections.upload.form.until-placeholder": "Jusque",
+  "submission.sections.upload.form.until-placeholder": "Jusqu'au",
 
   // "submission.sections.upload.header.policy.default.nolist": "Uploaded files in the {{collectionName}} collection will be accessible according to the following group(s):",
-  "submission.sections.upload.header.policy.default.nolist": "Les fichiers uploadés dans la Collection {{collectionName}} seront accessibles au(x) groupe(s) suivant(s) :",
+  "submission.sections.upload.header.policy.default.nolist": "Les fichiers téléchargés dans la collection {{collectionName}} seront accessibles au(x) groupe(s) suivant(s) :",
```

Comment:
"téléversés" would be better than "téléchargés" to translate "uploaded".

Reply:
@pilasou  see [above comment](https://github.com/DSpace/dspace-angular/pull/1580#discussion_r840840327)